src/config.cpp

```diff
@@ -947,6 +951,10 @@ void apply_config(std::unordered_map<std::string, std::string> &&vars) {
 
   string_restricted_f(vars, "gamepad"s, input.gamepad, platf::supported_gamepads());
 
+  bool_f(vars, "suppress_mouse", input.suppress_mouse);
+  bool_f(vars, "suppress_keyboard", input.suppress_keyboard);
+  bool_f(vars, "suppress_gamepad", input.suppress_gamepad);
```

Comment:
I think these should be named as follows:

- input.mouse
- input.keyboard
- input.controller (since input.gamepad is already used)

With the default value being enabled.

Reply:
That seems reasonable, and I can make that change. Would you prefer I add another commit, or amend the current one to keep it at just one commit?

Reply:
> Would you prefer I add another commit, or amend the current one to keep it at just one commit?

I would normally do an amend, but it's up to you. We will do a squash merge when we merge the PR so it will appear as a single commit anyway.

Reply:
@ReenigneArcher I've made the change, and kept it as a separate commit so it's easier to see what's changed from my last commit.